.gitignore

```diff
@@ -0,0 +1 @@
+./idea
```

Comment:
@ShoiraTa Thank you!

Actually, there is a commit made by @banlon-jones since he was the one who coded the HTML form in my project! 

![image](https://user-images.githubusercontent.com/92640684/144073506-e9ec14ea-fe47-4f6f-929c-4f5f264a7510.png)